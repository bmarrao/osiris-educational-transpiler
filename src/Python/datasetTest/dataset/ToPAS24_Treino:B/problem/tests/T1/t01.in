3 2
2 9 9 8 10 1
5 X 12 B
