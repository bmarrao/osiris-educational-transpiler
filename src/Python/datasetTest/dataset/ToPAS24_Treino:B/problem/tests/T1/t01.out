9 9
8 8 8 8 8 8 8 8 8
1 1 1 1 1 1 1 1 1 1
X X X X X
B B B B B B B B B B B B
